years.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if (grep -r "Copyright \+(c) \+2009-.*rultor.com" \
+  --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . \
+  | grep -v 2009-`date +%Y`) then
+    echo "Files above have wrong years in copyrights"
+    exit 1
+fi
+
+if (grep -r -L "Copyright \+(c) \+2009-.*rultor.com" \
+ --include=*.{java,xml,vm,groovy,txt,fml,properties} --exclude-dir "target" . \
+ | grep -v 2009-`date +%Y`) then
```

Comment:
@mkordas I am giving up. I've tried lots of different combinations - all of them worked well on my Mac OS 10.10 (bash 3.2) and Ubuntu 14.04 LTS.
Final try was just 
```
if (grep -r -L "Copyright \+(c) \+2009-.*rultor.com" \
 --include=*.{java,xml,vm,groovy,txt,fml,properties} --exclude-dir "target" .);\
  then
    echo "Files above must have copyright block in header"
    exit 1
fi

```
It's not working on travis (but also working on your environment). I have no idea, mabe tomorrow... Will be great thanks to you if you have any ideas or suggestions.